### PR TITLE
add tutorial for trading stellar lumens on kraken

### PR DESCRIPTION
**Pull request by nova8472**

### CODE REVIEW

**Comment by robertDurst** [comment]
Hey! Was checking this out and the link wasn't working for me -- did not see anything on the gitlab linked.

**Comment by nova8472** [comment]
@robertDurst I thank you for taking the time to check it out. I am not certain what the problem might have been. 

 The tutorial is in the Gitlab project's wiki page: https://gitlab.com/yumit.goog/stellar-kraken-tutorial/wikis/home 

Perhaps you had a look at the main page of the project? 

Or was it not accessible at all?

**Comment by robertDurst** [comment]
Ah cool, sorry missed that. Would you mind adding that link as source and then resolving conflict? Thanks!

File: TutorialsAndDocs.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 ## Tutorials And Documentation
 
+### [Trading Stellar lumens on Kraken](https://gitlab.com/yumit.goog/stellar-kraken-tutorial/wikis/home) - [source](https://gitlab.com/yumit.goog/stellar-kraken-tutorial/wikis/home)   
+A tutorial for buying and selling Stellar lumens on the Kraken.com exchange.  
+&nbsp;
+
+[Click here to learn how to list your project.](https://github.com/stellar/os-projects/tree/master#-how-to-get-your-project-listed)
```
Comment (robertDurst) [inline TutorialsAndDocs.md]: Would you mind removing this line?

File: TutorialsAndDocs.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 ## Tutorials And Documentation
 
+### [Trading Stellar lumens on Kraken](https://gitlab.com/yumit.goog/stellar-kraken-tutorial/wikis/home) - [source](https://gitlab.com/yumit.goog/stellar-kraken-tutorial/wikis/home)   
+A tutorial for buying and selling Stellar lumens on the Kraken.com exchange.  
+&nbsp;
+
+[Click here to learn how to list your project.](https://github.com/stellar/os-projects/tree/master#-how-to-get-your-project-listed)
+=======
```
Comment (robertDurst) [inline TutorialsAndDocs.md]: This line as well.

**Review by robertDurst — CHANGES_REQUESTED** [review]

**Comment by nova8472** [comment]
I am a little new to Github I am afraid. I am not certain if I did it quite right. 

Please, let me know if anything else needs fixing and thank you again for your guidance. :)

**Comment by robertDurst** [comment]
No worries, looks good now! Thanks!

**Review by robertDurst — APPROVED** [review]

**Pull request merged**